libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -73,6 +73,12 @@ export abstract class Bucket extends Resource {
    */
   public abstract addObject(key: string, body: string): void;
 
+  /**
+ * Add a file to the bucket from system folder
+ */
+
+  public abstract addFile(fileName: string, path: string): void;
```

Comment:
First parameter should be named `key` to align with the rest of our terminology/docs for `cloud.Bucket`
```suggestion
  public abstract addFile(key: string, path: string): void;
```

Reply:
Okay.